t/31-client_archive.t

```diff
@@ -27,16 +27,16 @@ subtest 'OpenQA::Client:Archive tests' => sub {
     is(-s "$limittest_path/limittest.tar.bz2", 2 * 1024 * 1024, 'limit test file is created')
       or note "dd output: $dd_output";
 
-    eval {
+    throws_ok {
         my %options = (
             archive => $destination,
             url => "/api/v1/jobs/$jobid/details",
             'asset-size-limit' => $limit,
             'with-thumbnails' => 1
         );
         my $command = $t->ua->archive->run(\%options);
-    };
-    is($@, '', 'Archive functionality works as expected would perform correctly') or always_explain $@;
+    }
+    qr//, 'Archive functionality works as expected would perform correctly' or always_explain $@;
```

Comment:
Use `dies_ok { ... }, $msg` instead of `throws_ok { ... } qr//, $msg`

Reply:
Maybe rather `lives_ok`, if i understsnd the original Code correctly

Reply:
yea it doesnt seem like dieing